x/auth/helpers/add_genesis_account_helper.go

```diff
@@ -0,0 +1,136 @@
+package helpers
+
+import (
+	"encoding/json"
+	"errors"
+	fmt "fmt"
+
+	"github.com/cosmos/cosmos-sdk/server"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	authvesting "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+)
+
+func AddGenesisAccount(path, moniker, amountStr string, accAddr sdk.AccAddress, appendAcct bool, vestingStart, vestingEnd int64, vestingAmtStr string) error {
```

Comment:
AddGenesisAccount shouldn't require a moniker. We should avoid passing homeDir if we can access it via `config`.
```suggestion
func AddGenesisAccount(config Context.Config, accAddr sdk.AccAddress, amountStr string, append bool, vestingStart, vestingEnd int64, vestingAmtStr string) error {
```